draco/asp/hard.lp

```diff
@@ -1,24 +1,98 @@
 % ====== Hard constraints ======
 
-% @hard(text_mark_without_text_channel) Text mark requires text encoding.
-violation(text_mark_without_text_channel) :-
-    attribute((mark,type),M,text),
-    not attribute(has_channel,M,text).
+% === Within Encodings ===
```

Comment:
I don't think these annotations work anymore with the new system, do they? Maybe we should split the hard constraints into multiple files.

Reply:
I thought these are just comments. Should I delete them instead?

Reply:
They made sense in Draco 1 where we didn't parse the file as we do now. If you look at the parsed `Block`s, you will see how things are parsed with this comment. 

We would need to add support for some annotation to the parser. We could for example add a new property `group` to the `Block` and then set it based on some special comment. 

For now, I would say remove it.